packages/teleport/src/Databases/AddDatabase/AddDatabase.test.tsx

```diff
@@ -46,28 +55,61 @@ describe('correct database add command generated with given input', () => {
 });
 
 test('correct tsh login command generated with local authType', () => {
-  render(<AddDialog {...props} />);
+  render(<AddDatabase {...props} />);
   const output = 'tsh login --proxy=localhost:443 --auth=local --user=yassine';
 
   expect(screen.queryByText(output)).not.toBeNull();
 });
 
 test('correct tsh login command generated with sso authType', () => {
-  render(<AddDialog {...props} authType="sso" />);
+  render(<AddDatabase {...props} authType="sso" />);
   const output = 'tsh login --proxy=localhost:443';
 
   expect(screen.queryByText(output)).not.toBeNull();
 });
 
+test('show correct steps when token attempt succeeded', () => {
+  render(
+    <AddDatabase {...props} authType="sso" attempt={{ status: 'success' }} />
+  );
+
+  expect(screen.queryByText('Step 1')).not.toBeNull();
+  expect(screen.queryByText('Step 2')).not.toBeNull();
+  expect(screen.queryByText('Step 3')).not.toBeNull();
+  expect(screen.queryByText('Step 4')).toBeNull();
+  expect(screen.queryByText('Step 5')).toBeNull();
+});
+
+test('show correct steps when token attempt failed', () => {
+  render(
+    <AddDatabase {...props} authType="sso" attempt={{ status: 'failed' }} />
+  );
+
+  expect(screen.queryByText('Step 1')).not.toBeNull();
+  expect(screen.queryByText('Step 2')).not.toBeNull();
+  expect(screen.queryByText('Step 3')).not.toBeNull();
+  expect(screen.queryByText('Step 4')).not.toBeNull();
+  expect(screen.queryByText('Step 5')).not.toBeNull();
+});
+
 test('render instructions dialog for adding database', () => {
-  render(<AddDialog {...props} />);
+  render(<AddDatabase {...props} />);
```

Comment:
we probably don't need this test anymore since we added snapshots with our storybook (it's pretty much the same thing just different props right?)